app/models/project_step.rb

```diff
@@ -67,6 +74,16 @@ def logs_count
     project_logs.count
   end
 
+  # Might be better as a filter
+  def schedule_ancestor=(ancestor)
+    self.scheduled_start_date = ancestor.scheduled_end_date if ancestor
```

Comment:
That would break this test: [spec/models/project_step_spec.rb#L105](https://github.com/sassafrastech/madeline_system/blob/193f1983ecc01b04587429d2ad624e6bf7e94b5d/spec/models/project_step_spec.rb#L105)

`super` is still called.